webpack.mix.js

```diff
@@ -1,6 +1,13 @@
 const mix = require('laravel-mix')
+const path = require('path')
```

Comment:
Codacy has a fix for the issue: [Replace `'path')` with `"path");`](https://app.codacy.com/gh/bluespada/go-telescope/pullRequest?prid=9653145)
```suggestion
const path = require("path");
```